build.cake

```diff
@@ -126,14 +126,6 @@ Task("PopulateRuntimes")
                 "win-x64",
                 "win-x86",
                 "win-arm64",
-                "ubuntu.14.04-x64",
```

Comment:
The list of runtime IDs was simplified in net8.0, which is why these were removed. More info here: https://learn.microsoft.com/en-us/dotnet/core/rid-catalog#known-rids